packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -11,3 +11,16 @@
   href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
   rel="stylesheet"
 />
+
+<style>
+  .sbdocs [class^="sbdocs"] {
+    color: #1d1d21;
+    font-family: "Public Sans", "-apple-system", "BlinkMacSystemFont",
+      "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell",
+      "Helvetica Neue", "Noto Sans Arabic", sans-serif;
+  }
+
+  .sbdocs .sbdocs-a {
+    color: #1662dd;
+  }
```

Comment:
😢  theming doesnt allow us to override this?

Reply:
Nope 😞 

SB-Docs doesn't have a theme interface, so this is their [preferred method](https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/theming.md#css-escape-hatches) for targeting the addon styles for now.